app/src/main/java/org/xwiki/android/sync/activities/base/BaseActivity.java

```diff
@@ -97,17 +116,31 @@ public void showProgressDialog(String message) {
         progress.show();
     }
 
+    /**
+     * Hide progress dialog if it currently visible.
+     */
     public void hideProgressDialog() {
         if (progress != null && progress.isShowing()) {
             progress.dismiss();
             progress = null;
         }
     }
 
+    /**
+     * @return {@link #progress}
+     */
     public ProgressDialog getProgressDialog() {
         return progress;
     }
 
+    /**
+     * Set default behaviour for menu items. Be careful if you want to override this method.
+     *
+     * @param item The menu item that was selected.
+     * @return boolean Return false to allow normal menu processing to
+     *         proceed, true to consume it here.
+     * @see #onCreateOptionsMenu
+     */
     @Override
```

Comment:
Methods with `@Override` automatically  inherit the javadoc so add something only if it's different from the overwritten method. See https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/JavaCodeStyle/#HDonotduplicateJavadoc for more details.

Reply:
Ok, I see

Reply:
fixed